mlir/test/python/ir/operation.py

```diff
@@ -668,6 +671,27 @@ def testKnownOpView():
         # CHECK: OpView object
         print(repr(custom))
 
+        # constant should map to an extension OpView class in the arithmetic dialect.
+        constant = module.body.operations[3]
+        # CHECK: <mlir.dialects.arith.ConstantOp object
+        print(repr(constant))
+        # CHECK: literal value 0
+        print("literal value", constant.literal_value)
```

Comment:
The extension is being registered successfully irrespective of this PR.

Reply:
Should these be comments in the file? I take these comments as to explaining what is being tested, which seems useful in the checked in test too.